lte/gateway/c/oai/tasks/nas/emm/Identification.c

```diff
@@ -244,9 +244,29 @@ int emm_proc_identification_complete(
          */
         emm_ctx_set_valid_imei(emm_ctx, imei);
       } else if (imeisv) {
-        /*
-         * Update the IMEISV
-         */
+        // Validate IMEISV
+        // Convert to string
+        char imeisv_str[MAX_IMEISV_SIZE + 1] = {0};
+        char imei_tmp[MAX_IMEI_SIZE]         = {0};
+        IMEISV_TO_STRING(imeisv, imeisv_str, MAX_IMEISV_SIZE + 1);
+        // Copy only 14 digits for validation
+        memcpy(imei_tmp, imeisv_str, (MAX_IMEI_SIZE - 1));
+        OAILOG_DEBUG(
+            LOG_NAS_EMM,
+            "EMM-PROC  - String imei "
+            "%s\n",
+            imei_tmp);
```

Comment:
no need to do this in two steps with `imeisv_str` and `imei_tmp`. See the other comment where you have the same constructs...